packages/account/package.json

```diff
@@ -43,7 +43,7 @@
         "mobx-react": "^7.5.1",
         "onfido-sdk-ui": "^11.0.0",
         "prop-types": "^15.7.2",
-        "qrcode.react": "^1.0.0",
+        "qrcode.react": "^3.1.0",
```

Comment:
to apply this on the package-lock file you need to run `npm run bootstrap:dev` from the root level and then commit the package-lock file.